conformance/eexp/basic_system_macros.ion

```diff
@@ -0,0 +1,91 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+// void
+
+(ion_1_1
+  (each (toplevel 1 ('#$:$ion:void') 2)
+        (toplevel 1 ('#$:$ion:0')    2)
+        (toplevel 1 ('#$:void')      2)
+        (produces 1                  2)))
+
+// TODO should this stay true?
+(ion_1_1
+  (toplevel 1 ('#$:$ion:void' true ('#$:void' false)) 2)
+  (produces 1 2))
+
+
+// values
+
+(ion_1_1
+  (each (toplevel 1 ('#$:$ion:values') 2)
+        (toplevel 1 ('#$:$ion:1')      2)
+        (toplevel 1 ('#$:values')      2)
+        (produces 1                    2))
+  (each (toplevel 1 ('#$:$ion:values' a) 2)
+        (toplevel 1 ('#$:$ion:1'      a) 2)
+        (toplevel 1 ('#$:values'      a) 2)
+        (produces 1                   a  2))
+  (each (toplevel 1 ('#$:$ion:values' a b) 2)
+        (toplevel 1 ('#$:$ion:1'      a b) 2)
+        (toplevel 1 ('#$:values'      a b) 2)
+        (produces 1                   a b  2)))
+
+
+// make_string
+
+// TODO Error when argument isn't text
+
+(ion_1_1
+  (each (toplevel ('#$:make_string'))
+        (toplevel ('#$:make_string' ""))
+        (toplevel ('#$:make_string' ''))
+        (produces ""))
+  // accepts both strings and symbols
+  (each (toplevel ('#$:make_string' "a" "b"))
+        (toplevel ('#$:make_string'  a   b ))
+        (toplevel ('#$:make_string' "a"  b ))
+        (toplevel ('#$:make_string'  a  "b"))
+        (produces "ab"))
+  // null.string and null.symbol act like ""
```

Comment:
Interesting; does `$0` also behave this way? I think I expected `null` arguments to be an error.

Reply:
Great question, I tend to forget about `$0`.  Heck, we also have absent symbols to deal with.

I cut https://github.com/amazon-ion/ion-docs/issues/324 to deal with that later.

Reply:
I spec'd nulls to act like `""` because their presence doesn't seem problematic enough to treat exceptionally.  This way all (known) symbol and string values are accepted without error.

(The cases you call out are arguably encoding defects, so I think they deserve more global handling.)